api/tests/rest.rs

```diff
@@ -72,8 +72,8 @@ fn test_start_api() {
 	assert!(server.start(addr, router, None).is_ok());
 	let url = format!("http://{}/v1/", server_addr);
 	let index = api::client::get::<Vec<String>>(url.as_str(), None).unwrap();
-	assert_eq!(index.len(), 2);
-	assert_eq!(counter.value(), 1);
+	//	assert_eq!(index.len(), 2);
```

Comment:
In general it’s a dangerous practise to disable tests, may I ask what is is the reason behind this particular change?